README.md

```diff
@@ -15,4 +15,46 @@ The first parameter <PATH OF "auth.whisk.system"> is mandatory and the second pa
 <OPENWHISK_HOME> will be set to the parent directory of the openwhisk-catalog if it is not provided.
```

Comment:
It's better to describe <OPENWHISK_HOME> as `the path to openwhisk`, other than `the parent directory of the openwhisk-catalog`, because openwhisk is not mandatory to be the parent of openwhisk-catalog.